### PR TITLE
Fix security section for Hubspot and GitLab

### PR DESCRIPTION
**Pull request by faonbr**

### CODE REVIEW

File: docs/integrations/hubspot/webhooks.md [hunk 1]
```diff
@@ -159,9 +159,9 @@ Verify that your local application receives the request and logs the correspondi
 
 ## Secure webhook requests {#security}
 
-The ngrok signature webhook verification feature allows ngrok to assert that requests from your HubSpot webhook are the only traffic allowed to make calls to your localhost app.
+The ngrok signature webhook verification feature allows ngrok to assert that requests from your Frame.io webhook are the only traffic allowed to make calls to your localhost app.
```
Comment (nijikokun) [inline docs/integrations/hubspot/webhooks.md:162]: Should be HubSpot

**Review by nijikokun — CHANGES_REQUESTED** [review]

File: docs/integrations/gitlab/webhooks.md [hunk 2]
```diff
@@ -155,6 +155,10 @@ Verify that your local application receives the request and logs the correspondi
 
 ## Secure webhook requests {#security}
 
+The ngrok signature webhook verification feature allows ngrok to assert that requests from your Frame.io webhook are the only traffic allowed to make calls to your localhost app.
```
Comment (russorat) [inline docs/integrations/gitlab/webhooks.md:158]:
```suggestion
The ngrok signature webhook verification feature allows ngrok to assert that requests from your Gitlab webhooks are the only traffic allowed to make calls to your localhost app.
```

File: docs/integrations/hubspot/webhooks.md [hunk 3]
```diff
@@ -159,9 +159,9 @@ Verify that your local application receives the request and logs the correspondi
 
 ## Secure webhook requests {#security}
 
-The ngrok signature webhook verification feature allows ngrok to assert that requests from your HubSpot webhook are the only traffic allowed to make calls to your localhost app.
+The ngrok signature webhook verification feature allows ngrok to assert that requests from your Hubspot webhook are the only traffic allowed to make calls to your localhost app.
```
Comment (russorat) [inline docs/integrations/hubspot/webhooks.md:162]:
```suggestion
The ngrok signature webhook verification feature allows ngrok to assert that requests from your HubSpot webhook are the only traffic allowed to make calls to your localhost app.
```

**Review by russorat — APPROVED** [review]
lgtm

**Pull request merged**